packages/next/src/resource.ts

```diff
@@ -86,23 +80,23 @@ async function lookupRoute(radixRouter: ReturnType<typeof createRouter>, req: Ne
   return makeApiRoute(req, route, pathParams)
 }
 
-export function createApiResourceRouter(core: GensekiCore) {
+export function createApiResourceRouter(core: GensekiCore, prefix: string = '/api') {
```

Comment:
make the second args to `object` instead of `string`. So that we can add more props in the future.